src/openms/include/OpenMS/CHEMISTRY/ModificationsDB.h

```diff
@@ -80,6 +80,16 @@ namespace OpenMS
       return db_;
     }
 
+    inline static ModificationsDB * getCustomInstance(OpenMS::String unimod_file, OpenMS::String psimod_file)
```

Comment:
hmm but we dont really want each AASequence to have its own ModificationsDB ptr and we dotn want to pass it around all the time. so maybe we can leave it be for the time being in the regular uses cases. However here we have to be very careful, what if a user wants to use a custom ModificationsDB, then some calls will use the custom db and some calls will use the default db (since they simply call `getInstance`) which could lead to some very hard to debug issues. 

In the light of the custom db I think we should (i) get rid of the static stuff or (ii) somehow allow the TOPP tool to change the ModificationsDB globally for all future calls, maybe by having a static member variable. However I am afraid of cases where parts of the code uses the modified db and other parts use the default db ...